CMakeLists.txt

```diff
@@ -701,41 +240,21 @@ if (NOT EVENT__DISABLE_THREAD_SUPPORT)
         find_package(Threads REQUIRED)
         if (NOT CMAKE_USE_PTHREADS_INIT)
             message(FATAL_ERROR
-                            "Failed to find Pthreads, set EVENT__DISABLE_THREAD_SUPPORT to disable")
+                    "Failed to find Pthreads, set EVENT__DISABLE_THREAD_SUPPORT to disable")
         endif()
 
         set(EVENT__HAVE_PTHREADS 1)
-        list(APPEND SRC_CORE evthread_pthread.c)
-        list(APPEND LIB_APPS ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
Use `Threads::Threads` here, available since CMake 3.1:
* https://cmake.org/cmake/help/v3.1/module/FindThreads.html

It will be easier to deal with `find_package` calls in `*Config.cmake`.

Reply:
Will do thanks.